src/node_url.cc

```diff
@@ -1283,7 +1283,9 @@ void URL::Parse(const char* input,
         }
         break;
       case kNoScheme:
-        cannot_be_base = base->flags & URL_FLAGS_CANNOT_BE_BASE;
+        if (has_base) {
```

Comment:
dunno / after https://github.com/nodejs/node/pull/14096 I think PRs are more intrisicly bundled if you need changes to land things, but might be mistaken